src/chinese_language_model.js

```diff
@@ -51,4 +51,8 @@ export default class ChineseLanguageModel extends LanguageModel {
     }
     typeFeaturesInitialized = true
   }
+
+  static getPunctuation () {
+    return ".,;:!?'\"(){}\\[\\]<>/\\\n\r\uFF0C\u3001\u3002\u300C\u300D\u300A\u300B"

```

Comment:
Can you just also add \u200C and  and \u200D (see alpheios-project/components#853)

Reply:
Added